docs/rst/tutorials/micro/custom_platforms/custom_platforms.rst

```diff
@@ -0,0 +1,341 @@
+  - ``rosidl_typesupport`` (`https://github.com/micro-ROS/rosidl_typesupport <https://github.com/micro-ROS/rosidl_typesupport>`_)
+  - ``rosidl_typesupport_microxrcedds`` (`https://github.com/micro-ROS/rosidl_typesupport_microxrcedds <https://github.com/micro-ROS/rosidl_typesupport_microxrcedds>`_)
+  - ``rosidl`` (`https://github.com/ros2/rosidl <https://github.com/ros2/rosidl>`_)
+  - ``rmw`` (`https://github.com/ros2/rmw <https://github.com/ros2/rmw>`_)
+  - ``rcl_interfaces`` (`https://github.com/ros2/rcl_interfaces <https://github.com/ros2/rcl_interfaces>`_)
+  - ``rosidl_defaults`` (`https://github.com/ros2/rosidl_defaults <https://github.com/ros2/rosidl_defaults>`_)
+  - ``unique_identifier_msgs`` (`https://github.com/ros2/unique_identifier_msgs <https://github.com/ros2/unique_identifier_msgs>`_)
+  - ``common_interfaces`` (`https://github.com/ros2/common_interfaces <https://github.com/ros2/common_interfaces>`_)
+  - ``test_interface_files`` (`https://github.com/ros2/test_interface_files <https://github.com/ros2/test_interface_files>`_)
+  - ``rmw_implementation`` (`https://github.com/ros2/rmw_implementation <https://github.com/ros2/rmw_implementation>`_)
+  - ``rcl_logging`` (`https://github.com/ros2/rcl_logging <https://github.com/ros2/rcl_logging>`_)
+  - ``ros2_tracing`` (`https://gitlab.com/ros_tracing/ros2_tracing <https://gitlab.com/ros_tracing/ros2_tracing>`_)
+  - ``micro_ros_utilities`` (`https://github.com/micro-ROS/micro_ros_utilities <https://github.com/micro-ROS/micro_ros_utilities>`_)
+
+5. Make sure that the following packages are removed or ignored:
+
+  - ``rosidl/rosidl_typesupport_introspection_cpp``
+  - ``rcl_logging/rcl_logging_log4cxx``
+  - ``rcl_logging/rcl_logging_spdlog``
+  - ``rclc/rclc_examples``
+  - ``rcl/rcl_yaml_param_parser``
+
+6. Build the micro-ROS workspace using colcon, your required ``colcon.meta`` and your custom compiler flags (using a CMake toolchain):
+
+.. code-block::
+
+  colcon build \
+      --merge-install \
+      --packages-ignore-regex=.*_cpp \
+      --metas my_colcon.meta \
+      --cmake-force-configure \
+      --cmake-clean-cache \
+      --cmake-args \
+      "--no-warn-unused-cli" \
+      --log-level=ERROR \
+      -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=OFF \
+      -DTHIRDPARTY=ON \
+      -DBUILD_SHARED_LIBS=OFF \
+      -DBUILD_TESTING=OFF \
+      -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
+      -DCMAKE_TOOLCHAIN_FILE=my_toolchain.cmake \
+      -DCMAKE_VERBOSE_MAKEFILE=ON;
+
+7. (Optional) Merge the generated ``.a`` libraries in a single one using `ar <https://man7.org/linux/man-pages/man1/ar.1.html>`_ utility.
+
+Integrating a custom build system
+---------------------------------
+
+At this point, the generated ``.a`` and ``include`` folder generated in the previous step shall be linked against a micro-ROS application.
+
+This procedure highly depends on the platform and the build system used but some approaches can be:
```

Comment:
> This procedure highly depends on the platform

This is repeated along the tutorial, maybe we can just say it once on the beginning and just assume it afterwards.

Reply:
9db93ea4ea7519166c7740a0c61a61b42f38fad8